src/RoleObjectFactory.js

```diff
@@ -173,7 +174,6 @@ function createAccessibilityObjectForRole(config) {
       break;
 
     case ROLES.GRID:
-    case ROLES.TREEGRID:
```

Comment:
treegrid needs its own subclass so that setters/getters are provided from applicable attributes (see https://www.w3.org/TR/wai-aria-1.1/img/rdf_model.svg) as well as providing overrides for keystroke handlers